drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -253,6 +254,22 @@ def _learner_raw_score(self):
         correct_count, total_count = self._get_item_stats()
         return correct_count / float(total_count)
 
+    @staticmethod
+    def _get_statici18n_js_url():
+        """
+        Returns the Javascript translation file for the currently selected language, if any.
```

Comment:
Can we mention here the use of `pkg_resources` as we're using it to fetch the translations' resources.

Reply:
☝️